components/DropDown.tsx

```diff
@@ -128,7 +155,7 @@ function DropDown({ municipalitiesName, placeholder, className }: Props) {
     } else {
       setShowDropDown(true)
     }
-    const filteredMunicipalities = sortedMunicipalities.filter((test) => test.toLowerCase().includes(value.toLowerCase()))
+    const filteredMunicipalities = search(value, sortedMunicipalities)
     setMunicipalities(filteredMunicipalities)
   }
 
```

Comment:
I'm unsure where to put these functions, if they should just sit here or be moved to a utils folder. One drawback right now is that they're being exported just for the sake of the unit tests.